cypress/utils/commands.js

```diff
@@ -62,6 +62,80 @@ Cypress.Commands.add('openWorkspaceDashboard', (workspaceName) => {
     .click();
 });
 
+// Command to measure component performance and compare with baseline
+// Exampele - cy.measureComponentPerformance({
+//   page: 'discover',
+//   componentTestId: 'docTable',
+//   eventName: 'onLoadSavedQuery',
+// });
+
+Cypress.Commands.add('measureComponentPerformance', (opts) => {
+  const { page, componentTestId, eventName, isDynamic = false } = opts;
+
+  cy.readFile('cypress/utils/performance_baselines.json').then((baselines) => {
```

Comment:
does the `performance_baselines.json` need to be a json file? If it was a `.js` file and the json converted into an object, i think that'll be much easier to work with

Reply:
makes sense, i just was going to comment how JSON makes it hard to comment, but looks like you already took care of that. thanks